File: worker.js
var redis = require("redis"),
        client1 = redis.createClient(),
        msg_count = 0;

    client1.on("message", function (channel, message) {
        console.log( channel + ": " + message);
    });

    client1.incr("did a thing");
    client1.subscribe(process.env.REDIS_CHANNEL);

File: worker.py
import redis, os, inspect

connection_pool = redis.ConnectionPool()
client = redis.Redis(connection_pool=connection_pool)
pubsub = client.pubsub()
pubsub.subscribe(os.environ.get('REDIS_CHANNEL'))

while(True):
    print "Listening"
    messages = pubsub.listen()
    if (messages is not None):
        print "WTF"
        for message in messages:
            print(message['data'])